eng/tools/generator/cmd/issue/issueCmd.go

```diff
@@ -257,6 +257,8 @@ func (c *commandContext) parseIssues(issues []*github.Issue) ([]request.Request,
 func (c *commandContext) buildConfig(requests []request.Request) (*config.Config, error) {
 	track1Requests := config.Track1ReleaseRequests{}
 	track2Requests := config.Track2ReleaseRequests{}
+	typespecRequests := config.TypeSpecReleaseRequests{}
+
 	for _, req := range requests {
 		switch req.Track {
 		case request.Track1:
```

Comment:
Shall we still need this setting track 1?   As we are cleaning up the track1 tool,  please ensure the code for handling track 1 release are removed as part of that effort.